frontend/express/public/core/app-management/templates/app-management.html

```diff
@@ -235,9 +276,9 @@ <h2 class="bu-mb-2" v-if="!newApp" >{{i18n('management-applications.plugins')}}
                         <cly-diff-helper :diff="changeKeys" @discard="onDiscard" @save="saveSettings" :disabled="v.invalid" :emitSaveWhenDisabled="true"></cly-diff-helper>
                     </validation-observer>
                 </cly-section>
-                <el-card class="box-card" v-if="!newApp" >
-                    <h3>{{i18n('common.integrate-sdks')}}</h3>
-                    <p>{{i18n('common.integrate-sdks-text')}}</p>
+                <el-card class="box-card color-cool-gray-100" v-if="!newApp" >
+                    <p>{{i18n('common.integrate-sdks')}}</p>
+                    <p><a href='https://code.count.ly' target='_blank'>Countly Code Generator </a>{{i18n('common.integrate-sdks-text')}}</p>
```

Comment:
Does this hurt whitelabeling @ar2rsawseen?

Reply:
There was a specific setting inside Settings -> Frontend -> Show Code Generator for SDK integration
so this part should only be displayed when that setting is true and should not be displayed when setting is false :)

Reply:
@hilalozturk1

Reply:
@hilalozturk1 just to speed things app, the setting should be in `countlyGlobal.config.code` so you can expose it to template and put in v-if